assets/js/src/segments/dynamic/dynamic_segments_filters/woocommerce.tsx

```diff
@@ -126,27 +126,27 @@ export const WooCommerceFields: React.FunctionComponent<Props> = ({ onChange, it
             type="number"
             min={0}
             value={item.number_of_orders_count || ''}
-            placeholder="count"
+            placeholder={MailPoet.I18n.t('wooNumberOfOrdersCount')}
             onChange={(event): void => compose([
               onChange,
               assign(item),
             ])({ number_of_orders_count: event.target.value })}
           />
-          <div>orders</div>
+          <div>{MailPoet.I18n.t('wooNumberOfOrdersOrders')}</div>
```

Comment:
Maybe you are not missing anything, and I'm the one that is missing something :-)

But I give what you suggested a try and could get to a half working solution where, to proceed, I would have to rely on the value of English words in the sentence and not the placeholders to add `<div>`s around them. That is obviously not an option, as those strings can change depending on the language. That is when I stopped and felt like it wasn't worth investing time to make this work in this particular PR and that we would probably need to come up with a more generic solution for strings that we want to interpolate with React components.

Reply:
I think this function with the `split` above should work as intended, please if there is something wrong, let me know. I am not sure how to make a generic solution from this.
```
        (match) => {
             if (match === '{type}') {
               return (
                 <Select />
               );
             }
            if (match === '{count}') {
               return (
                 <Input />
               );
             }
            if (match === '{days}') {
               return (
                 <Input />
               );
             }
             if ((typeof match === 'string') && match.trim().length > 1) {
               return (
                 <div key={match}>
                   {match}
                 </div>
               );
             }
             return null;
           }
```

Reply:
Thanks for sharing the example, @pavel-mailpoet. I got to this point but that is when I stopped. Unless I'm missing something, the example that you shared only works well for a single row. But in this case, we are splitting a sentence into two rows and that is when it gets tricky. We could split the string into two and then use the code above for each of the rows, but I don't know if that would be much better as the whole point is to let translators translate a single string and change the order of the input elements to match what makes more sense in their language. Do you see any solution to this that I'm seeing? If not, maybe we can proceed the way it is as @NeosinneR mentioned it in the ticket description as an option?

Also, @NeosinneR if we want to make this fully translatable maybe we should consider a different format for displaying the input fields? Maybe instead of interpolating them in the string, we create a row for each input field and use an individual string to describe each input field? Something like the example below but with a better wording:

```
Select the type of the comparison: <Select />
Select the number of orders: <Input />
Select the number of days: <Input />
```

Reply:
Oh, that is what I was missing. The sentence is split into 2 rows. Yeah, I don't think we can code that.